superset/db_engine_specs/hive.py

```diff
@@ -253,8 +253,9 @@ def handle_cursor(cls, cursor, query, session):
         last_log_line = 0
         tracking_url = None
         job_id = None
+        query_id = query.id
```

Comment:
minor nit: it might be easier to track stuff here if we use the query's client_id because that's how it's referred to on the client too.